### PR TITLE
Add OFAC sanction list

### PR DESCRIPTION
**Pull request by hildobby**
Brief comments on the purpose of your changes:

*For Dune Engine V2*
I've checked that:
General checks:
* [X] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [X] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [X] if adding a new model, I added a test
* [X] the filename is unique and ends with .sql
* [X] each sql file is a select statement and has only one view, table or function defined  
* [X] column names are `lowercase_snake_cased`
* [X] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [X] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [X] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [X] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [X] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [X] where block_time >= date_trunc("day", now() - interval '1 week')
* [X] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [X] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/addresses/sanctioned_addresses_ofac.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{{config(alias='sanctioned_addresses')}}
```
Comment (soispoke) [inline models/addresses/sanctioned_addresses_ofac.sql:1]: I suggest changing this model's name to models/addresses/ethereum/addresses_ethereum_ofac_sanctionned.sql

File: models/labels/ofac/labels_ofac_sanction.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{{config(alias='ofac_sanctioned')}}
```
Comment (soispoke) [inline models/labels/ofac/labels_ofac_sanction.sql:1]: I suggest changing this model's name to models/labels/labels_ofac_saction_ethereum.sql

File: macros/alter_table_properties.sql [hunk 3]
```diff
@@ -334,6 +334,15 @@ ALTER VIEW labels.nft SET TBLPROPERTIES('dune.public'='true',
                                         'dune.data_explorer.contributors'='["soispoke"]');
 {% endset %}
 
+{% set labels_ofac_sanction %}
```
Comment (soispoke) [inline macros/alter_table_properties.sql:337]: please also make changes here according to new models's names

**Comment by hildobby** [comment]
Think it should be good now, lmk @soispoke

**Comment by soispoke** [comment]
added the post hook config to expose models, now everything lgtm 👌

**Review by soispoke — APPROVED** [review]

**Pull request merged**